plugins/gatsby-transformer-garden-svg/gatsby-node.js

```diff
@@ -5,9 +5,89 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+const fetch = require('node-fetch');
 const { createNodeHelpers } = require('gatsby-node-helpers');
 const { optimize } = require('svgo');
```

Comment:
Are these imports used?

Reply:
Yeah, those imports are being used.

`createNodeHelpers` is used to return the `createNodeFactory` and `optimize` is used within `parseSvg`.